gm4_metallurgy/data/gm4_infinitas_shamir/functions/debug/clear.mcfunction

```diff
@@ -31,7 +31,7 @@ tag @s remove gm4_infinitas_delay_replace_water
 
 tag @s add gm4_has_empty_infinitas
 execute if score $lava_infinitas gm4_ml_data matches 1 run tag @s add gm4_has_lava_infinitas
-tag @s add gm4_has_powder_snow_infinitas
+execute if score $powder_snow_infinitas gm4_ml_data matches 1 run tag @s add gm4_has_powder_snow_infinitas
```

Comment:
At this point, this specific debug function is not needed, but may be useful for devs, in case they do something that breaks them?   Though since this can't really happen in the current state, they would have a large rewrite that would probably have their own debug functions.

The Calling Bell one is definitely applicable in that you can't easily spawn a Wandering Trader with llamas, similar to how you can't get Orbs from OoA with pneumas.  So imo, we should be picky about when we let the debug functions through, but not have a blanket "no".